### PR TITLE
fix: redirect for /docs

### PR DESCRIPTION
**Pull request by ethangj**

### CODE REVIEW

**Comment by hferentschik** [comment]
I guess this is a WIP, right?

File: content/en/docs/getting-started/_index.html [hunk 1]
```diff
@@ -2,7 +2,12 @@
 title="Getting Started"
 linkTitle="Getting Started"
 description="How to get up and running with Jenkins X"
-aliases="/getting-started"
+
+aliases = [
```
Comment (hferentschik) [inline content/en/docs/getting-started/_index.html:6]: :+1 nice one

File: content/en/docs/getting-started/_index.html [hunk 2]
```diff
@@ -2,7 +2,12 @@
...
+    "/getting-started",
+    "/docs",
```
Comment (hferentschik) [inline content/en/docs/getting-started/_index.html:8]: I think it is important to have w/ and w/o the trailing slash

**Comment by ethangj** [comment]
@hferentschik just needs the approval to merge, update title and commit message.

**Pull request merged**